torch/_dynamo/exc.py

```diff
@@ -819,6 +819,32 @@ def format_error_msg_verbose(
     return msg
 
 
+def format_frame_info(code: types.CodeType) -> str:
+    return (
+        f"{getattr(code, 'co_name', '<unknown>')} "
+        f"({getattr(code, 'co_filename', '<unknown>')} "
+        f"line {getattr(code, 'co_firstlineno', 0)})"
+    )
+
+
+def format_skip_frame_message(code: types.CodeType, reason: str) -> str:
+    frame_info = format_frame_info(code)
+    return (
+        f"torch.compile intentionally decided to skip the frame {frame_info} and fall back to eager.\n"
```

Comment:
I don't see a test that checks for this string - I think this might not be logged to the `graph_breaks` logging artifact (I believe it is logged to the `torch._dynamo` logger?). Can you make sure this is covered by the tests you added?